components/datetime/src/fieldset.rs

```diff
@@ -1013,7 +1007,7 @@ impl_zone_marker!(
     /// only a full time zone info can be formatted with this style.
     /// For example, [`TimeZoneInfo<AtTime>`] cannot be formatted.
     ///
-    /// ```compile_fail
+    /// ```compile_fail,E0271
```

Comment:
You can put the expected error number here? I've never seen that before but I love if it works.